doc/source/howtos/virt_sandbox.rst

```diff
@@ -325,7 +396,7 @@ To confirm that the user has been set up correctly, you can try to log in:
 
 .. code-block:: bash
 
-   root@proxysql:~# mysql -u sbuser -psbpass -h 127.0.0.1 -P 6033
```

Comment:
Since we are connected to ProxySQL node we can use 127.0.0.1 to connect ProxySQL DB.